tests/components/climate/conftest.py

```diff
@@ -21,3 +29,39 @@ def config_flow_fixture(hass: HomeAssistant) -> Generator[None]:
 
     with mock_config_flow("test", MockFlow):
         yield
+
+
+@pytest.fixture
+def integration_helper(hass: HomeAssistant, config_flow_fixture: None) -> Generator:
+    """Set up an integration."""
```

Comment:
I find the purpose of this fixture hard to understand from its name and docstring.
Maybe name the fixture "test_integration", and change the docstring to `"""Provide a mock integration for tests."""` or something like that?